modules/tinymce/src/plugins/media/test/ts/atomic/UpdateHtmlTest.ts

```diff
@@ -0,0 +1,37 @@
+import * as UpdateHtml from '../../../main/ts/core/UpdateHtml';
+import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { MediaData } from 'tinymce/plugins/media/core/Types';
+
+UnitTest.test('atomic.core.HtmlToDataTest', () => {
```

Comment:
This name is almost definitely wrong. It should end with `UpdateHtmlTest` and the prefix looks incorrect as well. I think this should be correct:
```
atomic.plugins.media.UpdateHtmlTest
```

Reply:
That'd likely be correct.